app/src/main/java/org/fossasia/openevent/general/order/Order.kt

```diff
@@ -47,5 +47,5 @@ data class Order(
     @Relationship("event")
     var event: EventId? = null,
     @Relationship("attendees")
-    var attendees: List<AttendeeId>? = null
+    var attendees: List<AttendeeId>
```

Comment:
@iamareebjamal This would fix things but I think we cannot remove the nullable property from here?

Reply:
No, this should be left as is. The only change required was in type of list

Reply:
Or you can add a default value of empty list. It'll solve all issues

Reply:
Done